### DISCUSSION TITLE
SSG for simple HTML and JS

### DISCUSSION THREAD

**Discussion by arcoyk** [vercel/community#628] [Help] [answered]
Hi,

I'd like to publish a web magazine with simple HTML and JS. Even Gasby.js seems extra for me. BUT, I want to use SSG for SEO (and WordPress is not enough for a mini-game on the top of the page). I came up with some ideas.

- Write SSG code on node.js
- Give up and use Gasby.js

Is there any other workaround for this?

Thanks

**Comment by leerob** [comment, marked as answer]
You could build your own custom build script using the [Build Output API](https://vercel.com/docs/build-output-api/v3) to "roll your own" framework.

**Reply by arcoyk** [reply]
Thanks, that's exactly what I needed.
